react/ProductCountPerPage.js

```diff
@@ -17,20 +18,40 @@ const ProductCountPerPage = () => {
     searchQuery
   )
 
+  const productsLoadedPercentage = Math.round(
+    (100 * products.length) / recordsFiltered
+  )
+
   if (products.length === 0) {
     return null
   }
 
+  const showingProducts =
+    productsLoadedPercentage !== 100
+      ? {
+          showingProductsId: 'store/search-result.showing-products',
+          showingProductsCountId: 'store/search-result.showing-products-count',
+        }
+      : {
+          showingProductsId: 'store/search-result.showing-all-products',
+          showingProductsCountId:
+            'store/search-result.showing-all-products-count',
+        }
```

Comment:
I think it's safer to check for 100, if `productsLoadedPercentage` is `NaN` it would render that all products were saw.

```suggestion
    productsLoadedPercentage === 100
      ? {
          showingProductsId: 'store/search-result.showing-all-products',
          showingProductsCountId:
            'store/search-result.showing-all-products-count',
        }
      : {
          showingProductsId: 'store/search-result.showing-products',
          showingProductsCountId: 'store/search-result.showing-products-count',
        }
```